playbooks/tmpfs/umount.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Linux | Delete the PKI directory
+  file:
+    path: "/{{ facts.tmpfs_volume_path }}/{{ facts.tmpfs_volume_name }}/"
+    state: absent
+  when: facts.ansible_system == "Linux"
+
+- name: MacOS | eject the ram disk
+  command: diskutil eject "/{{ facts.tmpfs_volume_path }}/{{ facts.tmpfs_volume_name }}/" || true
```

Comment:
What can be the reasons the eject fails? It seems that the `|| true` will silently ignore if it does?

Reply:
Yes, it does. I've separated the task and removed `|| true`. With this approach no errors expected.